src/annotation-engine/index.tsx

```diff
@@ -24,6 +25,7 @@ const AnnotationEngine: FC<AnnotationEngineProps> = forwardRef(
             annotations = [],
             backgroundImagePath,
             className,
+            drawingEvent = 'drag',
```

Comment:
Du coup ici tu peux utiliser ton `enum` tout neuf ;) 
```suggestion
            drawingEvent = DrawingEvent.DRAG,
```

Reply:
oopsie